### ISSUE TITLE
Requesting support for gp3 disk type

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-aws-cloudformation#150]
## Do you already have an issue opened with F5 support?
No

## Description
Requesting support for gp3 disk type. Currently the templates hard code gp2 with no option to supply as a variable. User must edit template disk block below example.

```
"Properties": {
"BlockDeviceMappings": [
{
"DeviceName": "/dev/xvda",
"Ebs": {
"DeleteOnTermination": "true",
"VolumeSize": "100",
"VolumeType": "gp2"
}
},
```

## Template
All AWS templates where BIG-IP is deployed with disk.

## Severity Level
3

**Comment by shyawnkarim** [comment]
Thanks for submitting this request. We are now tracking it internally with ID ESECLDTPLT-2566.

**Comment by shyawnkarim** [comment]
Closing due to age. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-aws-cloudformation-v2).
